backend/typescript/services/implementations/storageService.ts

```diff
@@ -0,0 +1,89 @@
+import { storage } from "firebase-admin";
+
+import IFileStorageService from "../interfaces/storageService";
+import logger from "../../utilities/logger";
+
+const Logger = logger(__filename);
+
+class FileStorageService implements IFileStorageService {
+  bucket: any;
```

Comment:
```suggestion
  bucket: string;
```

Reply:
It's an object, not a string. We need to add a dependency to strongly type it.

Reply:
Ah gotcha, we can leave it like this for now then